rusoto/rusoto
# run S3 integration tests against Ceph and Minio

* Allow to specify a custom endpoint in S3 integration_tests (localhost in our case).
* Run S3 integration tests against:
    * Ceph (latest)
    * Minio (latest and edge)
* use `-` instead of `_` in bucket names (Minio doesn't allow `_`)
* add feature gates to selectively disable tests not supported by  Ceph or Mino
* fix regression in custom endpoint support

There are still a few tests that don't work with Minio. I'll try to track the issues down and create separate pull requests to resolve them. In the meantime this should help to make sure no new regression are introduced when it comes to Ceph/Minio support.

related to #732
